manifests/0000_51_olm_01_operator_namespace.yaml

```diff
@@ -3,9 +3,10 @@ kind: Namespace
 metadata:
   name: openshift-cluster-olm-operator
   annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
```

Comment:
I think this label is related to hypershift and we would need to remove it. 

Ref: https://redhat-internal.slack.com/archives/C06KP34REFJ/p1731953839542759?thread_ts=1731946400.450489&cid=C06KP34REFJ

Reply:
Agree, fixed it.